app/src/main/java/org/listenbrainz/android/repository/preferences/AppPreferencesImpl.kt

```diff
@@ -44,9 +50,37 @@ import org.listenbrainz.android.util.Constants.Strings.USERNAME
 import org.listenbrainz.android.util.LinkedService
 import org.listenbrainz.android.util.TypeConverter
 
-class AppPreferencesImpl(private val context : Context): AppPreferences {
-    
+class AppPreferencesImpl(private val context: Context): AppPreferences {
     companion object {
+        private val gson = Gson()
+        private val blacklistMigration: DataMigration<Preferences> =
```

Comment:
Discussed on IRC to use other terms

Reply:
I don't think we need to change anything code-wise. Only the UI that the user interfaces with should not reflect it. Here, blacklist and whitelist signify what the preferences store.

Reply:
No @07jasjeet . We should avoid the usage like this here from the code as well.

Reply:
We decided to change this later